requirements-base.txt

```diff
@@ -42,6 +42,7 @@ phonenumberslite>=8.12.32
 Pillow>=10.2.0
 progressbar2>=3.41.0
 python-rapidjson>=1.4
+psutil>=5.9.2
```

Comment:
this is also included in tests requirements so you'll need to remove it from there